modules/nested-elements/assets/js/editor/nested-repeater/views/widget-repeater/empty.js

```diff
@@ -0,0 +1,91 @@
+import { useRef, useState } from 'react';
+
+export default function Empty( props ) {
+	// On container remove.
+	if ( props.container.view.isDisconnected() ) {
+		return null;
+	}
+
+	const containerHelper = $e.components.get( 'editor' ).defaultUtils().container,
+		addAreaElementRef = useRef(),
+		[ isRenderPresets, setIsRenderPresets ] = useState( false ),
+		onPresetSelected = ( preset, container ) => {
+			const options = {
+				createWrapper: false,
+			};
+
+			// Create new one by selected preset.
+			containerHelper.createContainerFromPreset( preset, container, options );
+		},
+		renderEmpty = () => {
+			const args = {
+				importOptions: {
+					target: props.container,
+				},
+			};
+
+			// Make droppable area. timeout means after the element is rendered, other React hooks causes errors.
+			setTimeout( () => {
```

Comment:
Use `useEffect` on top of the component.

Reply:
Handled.